v2/src/css/custom.css

```diff
@@ -519,3 +519,48 @@ html[data-theme="dark"] .markdown blockquote > *:last-child {
     bottom: 144px !important;
   }
 }
+
+/* Styles for collapsible elements */
+details {
+  position: relative;
+  transition: 0.3s, 0s border;
```

Comment:
The transition feels really strange especially since the text inside has no transition. It just pops in and then the rest of the detail component transitions in and out.

We should just remove the transition all together (both the detail and the ::before)

Reply:
its removed